src/ioredis-cluster/getClusterNodes.js

```diff
@@ -0,0 +1,70 @@
+const Redis = require('ioredis')
+const hash = require('object-hash')
+
+const redisNodesCache = {}
+module.exports = async function getClusterNodes(servers, options={}){
+
+    if(!Array.isArray(servers)){
+      servers = [servers]
+    }
+
+    for(const server of servers){
+      try{
+
+        const {
+          cache = true,
+          force = false,
+        } = options
+        const id = cache ? hash(server) : null
+        if(cache && !force && redisNodesCache[id]){
```

Comment:
same, it's not used in redis-ui, it's for loading nodes list and saving it somewhere, so in case you just filled one node, and this node failed, it can try others saved nodes, it was a part of my first implementation (first pull request used new Map() js object instead of hash)